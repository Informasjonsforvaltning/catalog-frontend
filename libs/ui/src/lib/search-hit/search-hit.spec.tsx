import React from 'react';
import { render } from '@testing-library/react';
import { SearchHit } from './';

describe('SearchHit', () => {
  it('should render SearchHit successfully', () => {
    const { baseElement } = render(
      <SearchHit
        searchHit={{
          id: 'bec18f0d-389e-4fa3-970d-23c1196cbe00',
          erPublisert: false,
          anbefaltTerm: {
            navn: {
              nb: 'test 2',
            },
          },
          definisjon: {
            tekst: {
              nb: 'anbefalt og definisjon',
            },
          },
          endringslogelement: {
            endringstidspunkt: '2023-01-16T14:29:28.768+01:00',
          },

          tildeltBruker: {
            id: 'John Doe',
          },
        }}
        catalogId={''}
      />,
    );
    expect(baseElement).toBeTruthy();
    expect(baseElement).toMatchSnapshot();
  });
});
