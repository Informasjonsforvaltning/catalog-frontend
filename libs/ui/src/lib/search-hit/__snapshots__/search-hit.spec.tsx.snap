// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchHit should render SearchHit successfully 1`] = `
<body>
  <div>
    <div
      class="container"
    >
      <div
        class="rowSpaceBetween"
      >
        <a
          href="undefined/bec18f0d-389e-4fa3-970d-23c1196cbe00"
        >
          <h2
            class="title"
          >
            test 2
          </h2>
        </a>
        <p
          class="greyFont"
        >
          Brødsmuler - Ikke klart i backend 
        </p>
      </div>
      <div
        class="metaData"
      >
        <p>
          Sist endret
            
        </p>
        <p>
          2023-01-16T14:29:28.768+01:00
        </p>
        <p
          class="dot"
        >
          •
        </p>
        <p>
          Utkast
        </p>
      </div>
      <div
        class="rowSpaceBetween"
      >
        <div
          class="metaData"
        >
          <p>
            Dette begrepet er under revisjon. Se utkast under versjoner. - Ikke klart i backend
          </p>
        </div>
        <p
          class="greyFont"
        >
          John Doe
        </p>
      </div>
      <p
        class="description"
      >
        anbefalt og definisjon
      </p>
    </div>
  </div>
</body>
`;
